Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml

```diff
@@ -37,6 +37,10 @@ ParserQuery: |
     , ASimAuthenticationPostgreSQL  (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationPostgreSQL'  in (DisabledParsers) ))
     , ASimAuthenticationSshd  (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationSshd' in (DisabledParsers) ))
     , ASimAuthenticationSu  (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationSu' in (DisabledParsers) ))
+    , ASimAuthenticationCiscoMeraki  (ASimAuthenticationDisabled or ('ExcludeASimAuthenticationCiscoMeraki' in (DisabledParsers) ))
+    , ASimAuditEventCiscoMeraki  (ASimAuthenticationDisabled or ('ExcludeASimAuditEventCiscoMeraki' in (DisabledParsers) ))
+    , ASimNetworkSessionCiscoMeraki  (ASimAuthenticationDisabled or ('ExcludeASimNetworkSessionCiscoMeraki' in (DisabledParsers) ))
+    , ASimWebSessionCiscoMeraki  (ASimAuthenticationDisabled or ('ExcludeASimWebSessionCiscoMeraki' in (DisabledParsers) ))
```

Comment:
@jayeshprajapaticrest would need to correct this one. 
Authentication union parser should have only Authentication schema specific entry. E.g. for Meraki we should be updating 4-4 files of ASim/vim type for 4 schemas. Total 8 files should get updated. Two for each schema.

Reply:
Removed other Cisco Meraki parsers from Authentication and added them in respective unifying parsers in respective PRs.